src/controllers/stallController.ts

```diff
@@ -201,6 +211,62 @@ async function bulkCreateStalls(req: Request, res: Response, next: NextFunction)
   }
 }
 
+// Upload csv file to import data
+const csvUpload = multer({
+  storage: multer.memoryStorage(),
+  fileFilter: generateFileFilter(/csv/),
+  limits: { fileSize: MAX_CSV_SIZE },
+});
+
+/**
+ * Csvfile needs to have stall attributes as its header on the first row.
+ *
+ * Example of csv file:
+ * name,description,contactNo,unitNo,hawkerCentreId
```

Comment:
The function can already take in `openingHours` in the csv but it has to be in the correct json format. As of now, no validation is done for `openingHours`.  It may be good to discuss a simpler format for `openingHours` for the csv import. 

Gonna merge this in first as the bare functionality is there.